1568382792555	Sync.LogManager	DEBUG	Flushing file log
1568382792555	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1568382792560	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1568382792578	Sync.LogManager	DEBUG	Log cleanup threshold time: 1567518792578
1568382792607	Sync.LogManager	DEBUG	Done deleting files.
1568382819854	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1568382819854	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1568382819857	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1568382820854	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1568382829789	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1568382829789	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1568382829796	Sync.Service	DEBUG	User-Agent: Firefox/69.0 (Linux x86_64) FxSync/1.71.0.20190827005903.desktop
1568382829796	Sync.Service	INFO	Starting sync at 2019-09-13 19:53:49 in browser session 0eDuYaQekR5c
1568382829796	Sync.Service	DEBUG	In sync: should login.
1568382829797	Sync.Service	INFO	User logged in successfully - verifying login.
1568382829797	Sync.BrowserIDManager	DEBUG	unlockAndVerifyAuthState already has (or can fetch) sync keys
1568382829798	Sync.Status	DEBUG	Status.login: error.login.reason.network => success.status_ok
1568382829798	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1568382829798	Sync.Service	DEBUG	Fetching unlocked auth state returned success.status_ok
1568382829798	Sync.BrowserIDManager	INFO	Getting keys
1568382829798	Sync.BrowserIDManager	INFO	Getting an assertion from: https://token.services.mozilla.com/1.0/sync/1.5
1568382829799	FirefoxAccounts	DEBUG	enter getAssertion()
1568382829804	FirefoxAccounts	DEBUG	got keyPair
1568382829805	FirefoxAccounts	DEBUG	getCertificateSigned: true true
1568382829808	Services.Common.RESTRequest	TRACE	local time, offset: 1568382829807, 0
1568382829815	Services.Common.RESTRequest	DEBUG	POST request to https://api.accounts.firefox.com/v1/certificate/sign
1568382829815	Services.Common.RESTRequest	TRACE	HTTP Header authorization: ***** (suppressed)
1568382829815	Services.Common.RESTRequest	TRACE	HTTP Header content-type: application/json
1568382829815	Services.Common.RESTRequest	TRACE	HTTP Header accept-language: en-US, en
1568382829815	Services.Common.RESTRequest	DEBUG	POST Length: 888
1568382829815	Services.Common.RESTRequest	TRACE	POST Body: {"publicKey":{"algorithm":"DS","y":"6067B7CDF7E427C1E9B201C84D8565BBCD1D69D66028F568C9EFB6CE5BC4C5C548E1AA2D6DCB4478ABF090D065E507355D6005F394E47B4A0B989CC5FF4B10F25CA04F5E94612032B7ECC0D052A0638AA3D520491CE593E123448548F8D906DB070B159E98D9F866C1B9946FD6F8D8686281C848E96F6E7C948CB0F7ED29A4D1","p":"FF600483DB6ABFC5B45EAB78594B3533D550D9F1BF2A992A7A8DAA6DC34F8045AD4E6E0C429D334EEEAAEFD7E23D4810BE00E4CC1492CBA325BA81FF2D5A5B305A8D17EB3BF4A06A349D392E00D329744A5179380344E82A18C47933438F891E22AEEF812D69C8F75E326CB70EA000C3F776DFDBD604638C2EF717FC26D02E17","q":"E21E04F911D1ED7991008ECAAB3BF775984309C3","g":"C52A4A0FF3B7E61FDF1867CE84138369A6154F4AFA92966E3C827E25CFA6CF508B90E5DE419E1337E07A2E9E2A3CD5DEA704D175F8EBF6AF397D69E110B96AFB17C7A03259329E4829B0D03BBC7896B15B4ADE53E130858CC34D96269AA89041F409136C7242A38895C9D5BCCAD4F389AF1D7A4BD1398BD072DFFA896233397A"},"duration":21600000}
1568382829820	Services.Common.RESTRequest	TRACE	onStartRequest: POST https://api.accounts.firefox.com/v1/certificate/sign
1568382829820	Services.Common.RESTRequest	TRACE	Channel for POST https://api.accounts.firefox.com/v1/certificate/sign returned status code 2152398878
1568382829820	Services.Common.RESTRequest	DEBUG	POST https://api.accounts.firefox.com/v1/certificate/sign failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1568382829820	Services.Common.RESTRequest	TRACE	POST body: 
1568382829820	Hawk	WARN	hawk request error: [Exception... "NS_ERROR_UNKNOWN_HOST"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487"  data: no] Stack trace: onStopRequest()@resource://services-common/rest.js:487
1568382829821	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 623"  data: no] Stack trace: get status()@resource://services-common/rest.js:623
request()@resource://services-common/hawkclient.js:239
1568382829821	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 638"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:638
request()@resource://services-common/hawkclient.js:247
1568382829821	Hawk	DEBUG	(Response) /certificate/sign: code: null - Status text: null
1568382829821	Services.Common.RESTResponse	TRACE	Processing response headers.
1568382829821	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 670"  data: no] Stack trace: get headers()@resource://services-common/rest.js:670
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:316
request()@resource://services-common/hawkclient.js:255
1568382829822	Services.Common.RESTResponse	TRACE	Processing response headers.
1568382829822	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 670"  data: no] Stack trace: get headers()@resource://services-common/rest.js:670
_maybeNotifyBackoff()@resource://services-common/hawkclient.js:316
request()@resource://services-common/hawkclient.js:256
1568382829822	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status text: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatusText]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get statusText :: line 638"  data: no] Stack trace: get statusText()@resource://services-common/rest.js:638
_constructError()@resource://services-common/hawkclient.js:122
request()@resource://services-common/hawkclient.js:261
1568382829823	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 623"  data: no] Stack trace: get status()@resource://services-common/rest.js:623
_constructError()@resource://services-common/hawkclient.js:123
request()@resource://services-common/hawkclient.js:261
1568382829823	Services.Common.RESTResponse	DEBUG	Caught exception fetching HTTP status code: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.responseStatus]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get status :: line 623"  data: no] Stack trace: get status()@resource://services-common/rest.js:623
_constructError()@resource://services-common/hawkclient.js:124
request()@resource://services-common/hawkclient.js:261
1568382829823	Services.Common.RESTResponse	TRACE	Processing response headers.
1568382829823	Services.Common.RESTResponse	DEBUG	Caught exception processing response headers: [Exception... "Component returned failure code: 0x80040111 (NS_ERROR_NOT_AVAILABLE) [nsIHttpChannel.visitResponseHeaders]"  nsresult: "0x80040111 (NS_ERROR_NOT_AVAILABLE)"  location: "JS frame :: resource://services-common/rest.js :: get headers :: line 670"  data: no] Stack trace: get headers()@resource://services-common/rest.js:670
_constructError()@resource://services-common/hawkclient.js:130
request()@resource://services-common/hawkclient.js:261
1568382829823	FirefoxAccounts	ERROR	error POSTing /certificate/sign: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568382829824	FirefoxAccounts	ERROR	HAWK.signCertificate error: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568382829824	Sync.BrowserIDManager	ERROR	Non-authentication error in _fetchTokenForUser: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568382829824	Sync.Status	DEBUG	Status.login: success.status_ok => error.login.reason.network
1568382829824	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1568382829824	Sync.BrowserIDManager	INFO	Failed to fetch the cluster URL: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568382829825	Sync.Service	DEBUG	verifyLogin failed: {"error":{},"errorString":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 487\"  data: no]","message":null,"code":null,"errno":null}
1568382829825	Sync.Status	DEBUG	Status.login: error.login.reason.network => error.login.reason.network
1568382829825	Sync.Status	DEBUG	Status.service: error.login.failed => error.login.failed
1568382829825	Sync.ErrorHandler	ERROR	Sync encountered a login error
1568382829826	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1568382829827	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1568382829829	Sync.Service	DEBUG	Exception calling WrappedLock: Error: Login failed: error.login.reason.network(resource://services-sync/service.js:984:15) JS Stack trace: onNotify@service.js:984:15
1568382829829	Sync.Service	DEBUG	Not syncing: login returned false.
